presto-main/src/test/java/io/prestosql/server/ui/TestWebUi.java

```diff
@@ -433,11 +434,11 @@ private static void testAlwaysAuthorized(URI baseUri, OkHttpClient authorizedCli
 
         assertOk(authorizedClient, getValidApiLocation(baseUri));
 
-        assertRedirect(authorizedClient, getLoginHtmlLocation(baseUri), getUiLocation(baseUri));
+        assertRedirect(authorizedClient, getLoginHtmlLocation(baseUri), getUiLocation(baseUri), false);
```

Comment:
Why?

Reply:
The proxy test doesn't go over https so the protocol authentication does not work